docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
+                </Example>
 
-                {/* Vertical Align */}
-                <TableSampleVerticalAlign />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Disable
+                    </Heading>
 
-                {/* onClick Event Handler Samples */}
-                <TableSampleOnClick />
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s row or cell can be disabled.
+                    </Typography>
+                </MarkdownContainer>
 
-                {/* Responsive Width */}
-                <TableSampleResponsiveWidth />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleDisabled').default}
+                >
+                    <TableSampleDisabled />
+                </Example>
 
-                {/* Drag and Drop Row Reordering */}
-                <TableSampleDragAndDropRowReordering />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Text Align
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s row or cell&rsquo;s text alignment can be changed.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleTextAlign').default}
+                >
+                    <TableSampleTextAlign />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Vertical Align
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s row or cell&rsquo;s vertical alignment can be changed.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleVerticalAlign').default}
+                >
+                    <TableSampleVerticalAlign />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Table Header Cell onClick Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table header cell can handle an onClick event.
```

Comment:
NitpickNAB: `onClick` should maybe be surrounded by `<code> .... </code>` ...?

Reply:
Fixed.

Reply:
Looks good.  Resolving.